src/html/htmlwin.cpp

```diff
@@ -1184,7 +1184,31 @@ void wxHtmlWindow::OnPaint(wxPaintEvent& WXUNUSED(event))
     }
 }
 
+void wxHtmlWindow::OnFocusEvent(wxFocusEvent& event)
+{
+    event.Skip();
 
+    // Redraw selection, because its background colour depends on
+    // whether the window has keyboard focus or not.
+
+    if ( !m_selection || m_selection->IsEmpty() )
+        return;
+
+    const wxHtmlCell* fromCell = m_selection->GetFromCell();
+    const wxHtmlCell* toCell = m_selection->GetToCell();
+    wxCHECK_RET(fromCell || toCell,
+                "Unexpected: selection is set but cells are not");
+    if ( !fromCell )
+        fromCell = toCell;
+    if ( !toCell )
+        toCell = fromCell;
+
+    const wxPoint topLeft = fromCell->GetAbsPos();
+    const wxPoint bottomRight = toCell->GetAbsPos() +
+        wxSize(toCell->GetWidth(), toCell->GetHeight());
+    RefreshRect(wxRect(CalcScrolledPosition(topLeft),
```

Comment:
Oops, no, sorry, further testing actually shows that this is wrong: there is no guarantee that the selection is entirely inside this rect, you really need to make a union of the cells of all rectangles between `fromCell` and `toCell` to be correct.

To see the problem: run `samples/html/test`, scroll down until you see
```
This is new par? 
We switched to BOLD 
This is new paragraph Bold is off now. 
new par 
```
and select all 4 lines, then click on the text control below (or on an another application window): you will see that "ew par" in the first line (as well as big parts of the 2nd and 3rd lines) remains blue because it's beyond the right bound of the bottom cell.

Reply:
Fixed.